### ISSUE TITLE
After reinstalling gam tried to use "gam use project" but receving JWT signature error

### ISSUE THREAD

**Issue by mrPsycho** [jay0lee/GAM#1453]
Full steps to reproduce the issue:
1. Deleted old version by `rm -rf ~/gam`
2. Installed new by `bash <(curl -s -S -L https://git.io/install-gam)`
3. Tried to re-use previosly created project by `gam use project`

Expected outcome (what are you trying to do?):
It will work as expected.

Actual outcome (what errors or bad behavior do you see instead?):
```
mrpsycho@DESKTOP-L6B9C6M:~/bin/gam$ gam user y.n check serviceaccount
Computer clock status:
 Your system time differs from admin.googleapis.com by less than 1 second   PASS
Service Account Private Key Authentication:
 Authenticating...invalid_grant: Invalid JWT Signature.                     FAIL

ERROR: Invalid private key in oauth2service.json. Please delete the file and then
recreate with "gam create project" or "gam use project"
mrpsycho@DESKTOP-L6B9C6M:~/bin/gam$ gam user y.n@domain.com check serviceaccount
Computer clock status:
 Your system time differs from admin.googleapis.com by less than 1 second   PASS
Service Account Private Key Authentication:
 Authenticating...invalid_grant: Invalid JWT Signature.                     FAIL

ERROR: Invalid private key in oauth2service.json. Please delete the file and then
recreate with "gam create project" or "gam use project"  
```

`gam info domain` works as expected.

**Comment by lonix86** [comment]
At the moment the only way is to manually download the JSON keys again from the project in GCP, the keys generated by the create project command are in fact not working.

**Comment by mrPsycho** [comment]
oh, thank you. 

re-downloaded json's and yes, it passed.

**Comment by jay0lee** [comment]
can anyone who has seen this issue confirm they live in a timezone that is ahead of UTC?

**Comment by lonix86** [comment]
I'm in GMT+2

**Comment by jay0lee** [comment]
Thanks @lonix86 

I just pushed out 6.11 with this fix. If someone who saw this issue has the bandwidth I'd appreciate confirmation it's resolved in 6.11.

**Comment by lonix86** [comment]
thanks to you @jay0lee for your excellent work, GAM for me and a lot of people like me is just fantastic (I speak to you as a Google Partner and Reseller, with thousands of customers)
